README.md

```diff
@@ -4,22 +4,38 @@ This code simulates proton-electron collisions using Pythia8 and saves in HepMC
 
 ## Installation
 
-To run this code, you need to install [Pythia8](https://pythia.org/), a program for the generation of high-energy physics collision events. You can download Pythia8 using:
+1) Get and run [eic-shell](https://eic.github.io/tutorial-setting-up-environment/02-eic-shell/index.html)
+
+2) Clone and make Pythia8.306
 ```
-git clone git@gitlab.com:Pythia8/releases.git --branch pythia8XXX --single-branch
+mkdir Pythia8306
+cd Pythia8306
+git clone https://gitlab.com/Pythia8/releases.git --branch pythia8306 --single-branch
+cd releases
+./configure --with-hepmc3
+make
+cd bin
+export LD_LIBRARY_PATH=$(pythia8-config --libdir):${LD_LIBRARY_PATH}
```

Comment:
Important that we don't pick up the system or eic-shell /usr/local version of `pythia8-config`.
```suggestion
export LD_LIBRARY_PATH=$(./pythia8-config --libdir):${LD_LIBRARY_PATH}
```